src/common/modules/popup/notice/NoticePopup.vue

```diff
@@ -60,10 +60,21 @@ export default {
             }
         };
 
+        const getNoticeBoard = async (): Promise<string> => {
+            try {
+                const { results } = await SpaceConnector.client.board.board.list();
+                return results.filter(d => d.name === 'Notice')[0]?.board_id ?? '';
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                return '';
+            }
+        };
```

Comment:
```suggestion
        const getNoticeBoard = async (): Promise<string | undefined> => {
            try {
                const { results } = await SpaceConnector.client.board.board.list();
                return results.filter(d => d.name === 'Notice')[0]?.board_id;
            } catch (e) {
                ErrorHandler.handleError(e);
                return undefined;
            }
        };
```

이렇게 하고 아래 getPostList에서 없을 경우 early return 하는 건 어떨까요?
웬만하면 있겠지만...그래도 방어코드를 위하야
```typescript
const noticeBoard = await getNoticeBoard();
if (!noticeBoard) return;
```